.mvn/maven.config

```diff
@@ -1,2 +1,4 @@
 -Pconsume-incrementals
 -Pmight-produce-incrementals
+-Pconsume-incrementals
```

Comment:
Duplicate profiles

Reply:
Thanks for catching that!  Fixed in ead0c5815dc72241c6d15994b49bd8556cfe83c5